lutris/runners/steam.py

```diff
@@ -235,6 +239,21 @@ def get_steamapps_dirs(self):
                     dirs.append(path)
                 i += 1
 
+        # New Custom dirs
+        if library_config := self.get_library_config():
+            paths = []
+            for entry in library_config.values():
+                if "mounted" in entry:
+                    if entry.get("path") and entry.get("mounted") == "1":
```

Comment:
Does lutris _really_ care if the directory is mounted to steam or not? 
If the folder is empty of steam games, then games aren't added. The only time I've been able to end up with a non-empty steam library folder that's not mounted is when I switch from the beta back to the stable version of steam.

@tannisroot @strycore @ashblk @AlexanderRavenheart

Reply:
if it's just a matter of fixing the linting issue, don't worry about it and don't change the behavior only to make pylint happy.
If you feel this path is work then I'll go ahead and merge it.

Reply:
It works as is for the scope of the PR. If this looks good for everyone else, I'm good with it being merged as is